tmt/steps/prepare/ansible.py

```diff
@@ -35,15 +37,16 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin):
     # Supported keys
     _keys = ["playbook", "extra-args"]
 
-    def __init__(self, step, data):
+    def __init__(self, step: str, data: TypedDict) -> None:
```

Comment:
This is not how it works @guy9050, TypedDict needs to be defined for the particular usage:

https://adamj.eu/tech/2021/05/10/python-type-hints-how-to-use-typeddict/

So you need to subclass TypedDict and defined the expect dictionary keys with they expected type